pkg/server/container_create_windows.go

```diff
@@ -166,8 +171,28 @@ func (c *criService) CreateContainer(ctx context.Context, r *runtime.CreateConta
 
 	log.G(ctx).Debugf("Container %q spec: %#+v", id, spew.NewFormatter(spec))
 
-	snapshotterOpt := snapshots.WithLabels(config.Annotations)
+	// If the config field is specified, set the snapshotter label to reuse the pods
+	// scratch space. This only affects LCOW at the moment.
+	val, ok := config.Annotations["containerd.io/snapshot/io.microsoft.container.storage.reuse-scratch"]
+	if rhcso.ShareScratch || (ok && val == "true") {
+		config.Annotations["containerd.io/snapshot/io.microsoft.container.storage.reuse-scratch"] = "true"
```

Comment:
Are these keys not defined anywhere?

Reply:
They're defined in the lcow snapshotter and not exported (although we could)

Reply:
Np its ok